pyproject.toml

```diff
@@ -50,7 +50,7 @@ pkginfo = "^1.9.4"
 platformdirs = "^3.0.0"
 pyproject-hooks = "^1.0.0"
 requests = "^2.26"
-requests-toolbelt = ">=0.9.1,<2"
+requests-toolbelt = ">=1,<2"
```

Comment:
Could you use the `^1.0.0` marker, please? It will mean the same in a nicer way.

Reply:
Sure, updated!